### PR TITLE
Info on public pipelines

### PR DESCRIPTION
**Pull request by plaindocs**
Keeping it simple.

### CODE REVIEW

File: pages/pipelines/public_pipelines.md.erb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Public Pipelines
+
+If you're working on an open source project, and want the whole world to be able to see your builds, you can make your pipeline public.
```
Comment (karensawrey) [inline pages/pipelines/public_pipelines.md.erb:3]:
```suggestion
If you're working on an open-source project, and want the whole world to be able to see your builds, you can make your pipeline public.
```

File: pages/pipelines/public_pipelines.md.erb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+- Pipeline build pages
+- Pipeline build logs
+- Pipeline build artefacts
+- Pipeline build environment config
+- Agent version and name
+
+## Make a pipeline public using the UI
+
+Make a pipeline public in the Pipeline settings:
```
Comment (karensawrey) [inline pages/pipelines/public_pipelines.md.erb:17]:
```suggestion
Make a pipeline public in the Pipeline Settings:
```

**Review by karensawrey — APPROVED** [review]
LGTM, save for 2 nitpicking suggestions (the 2nd one might be erroneous, the context is a bit unclear).

File: pages/pipelines/public_pipelines.md.erb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
```
Comment (plaindocs) [inline pages/pipelines/public_pipelines.md.erb:17]:
```suggestion
Make a pipeline public in the _Pipeline Settings_:
```

I'll go one further

**Pull request merged**